### DISCUSSION TITLE
【求助】绝区零50级之前该刷什么

### DISCUSSION THREAD

**Discussion by rainbowwarmth** [share121/inter-knot#1874] [其他]
绝区零50级之前该刷什么

**Comment by cyberdragonvein** [comment]
先给主c刷一套主词条对的s级4+2，然后给辅助刷只需要6号位是s级主词条对的4+2，然后多刷丁尼和经验书，突破材料等到50级以后再打

**Comment by 834398220** [comment]
50级的副本和满级副本掉落物数量差距最小的就是丁尼，50级前，刷丁尼最划算！

**Comment by baolovey** [comment]
当然是刷丁尼!等50级了再刷突破材料
